content/docs/rfcs/44/README.md

```diff
@@ -0,0 +1,275 @@
+
+44/WAKU2-DANDELION does not protect against targeted deanonymization attacks.
+
+### Non-Dandelion Peers
+
+Stem relays receiving messages can either be in stem state or in fluff state themselves.
+They might also not support 44/WAKU2-DANDELION,
+and interpret the message as classical [19/WAKU2-LIGHTPUSH](/spec/19/),
+which effectively makes them act as fluff state relays without introducing random delay.
+While such peers lower the overall anonymity properties,
+the [Dandelion++ paper](https://arxiv.org/abs/1805.11060)
+showed that including those peers yields more anonymity compared to excluding these peers.
+
+## Future Analysis
+
+The following discusses potential relaxations in favour of reduced latency,
+as well as their impact on anonymity.
+This is still work in progress and will be elaborated on in future versions of this document.
+
+Generally, there are several design choices to be made for the stem phase of a Dandelion-based specification:
+
+1) the probability of continuing the stem phase, which determines the expected stem lengh,
+2) the out degree in the stem phase, which set to 1 in this document (also in the Dandelion papers),
+3) the rate of re-selecting stem relays among all gossipsub mesh peers (for a given pubsub topic), and
+4) the mapping of incoming connections to outgoing connections.
+
+### Bound Stem Length
+
+Choosing $q = 0.2$, 44/WAKU2-DANDELION has an expected stem length of 5 hops,
+Assuming $100ms$ added delay per hop, the stem phase adds around 500ms delay on average.
+
+There is a possibility for the stem to grow longer,
+but some applications need tighter bounds on latency.
+
+While fixing the stem length would yield tighter latency bounds,
+it also reduces anonymity properties.
+A fixed stem length requires the message to carry information about the remaining stem length.
+This information reduces the uncertainty of attackers
+when calculating the probability distribution assigning each node a probability for having sent a specific message.
+We will quantify the resulting loss of anonymity in future versions of this document.
+
+### Stem Relay Selection
+
+In its current version, 44/WAKU2-DANDELION nodes default to fluff state
+if the random stem relay selection yields at least one peer that does not support [19/WAKU2-LIGHTPUSH](/spec/19/) (which is the stem protocol used in 44/WAKU2-DANDELION).
+If nodes would reselect peers until they find peers supporting [19/WAKU2-LIGHTPUSH](/spec/19/),
+malicious nodes would get an advantage if a significant number of honest nodes would not support [19/WAKU2-LIGHTPUSH](/spec/19/).
+Even though this causes messages to enter fluff phase earlier,
+we choose the trade-off in favour of protocol stability and sacrifice a bit of anonymity.
+(We will look into improving this in future versions of this document.)
+
+### Random Delay in Fluff Phase
+
+The random delay in the fluff phase MAY be set to 0 to achieve lower latency.
```

Comment:
I'm a bit confused by this delay, it's said earlier that no modifications to gossipsub are required, but this delay would be a modification to gossipsub?
Also, it's not explained why it's necessary

Last point, iirc each waku message contains the originator timestamp, seems to me that this break dandelion, since I can know if you are the originator if now - timestamp < 100ms

Reply:
Already merged, but fwiw I agree to leave this out of the spec for now. :)

Also, re:

>  iirc each waku message contains the originator timestamp

This timestamp is optional, precisely because of the security risks. Also, I had the idea that this timestamp should tick on something like 500ms boundaries, which will be useful enough to use it for ordering, but too coarse to be effectively used in timing attacks. I strongly suggest that we make this part of a recommendation in specs soon.